pyproject.toml

```diff
@@ -46,7 +46,7 @@ pandas = ">=1.3.0"
 httpx = ">=0.23.0"
 attrs = ">=20.1.0"
 python-dateutil = "^2.8.0"
-pyarrow = "^14.0.2"
+pyarrow = ">=15.0.2"
```

Comment:
To address a time unit issue https://dominodatalab.atlassian.net/browse/DOM-56685